command/pr_create.go

```diff
@@ -203,11 +202,26 @@ func prCreate(cmd *cobra.Command, _ []string) error {
 			return fmt.Errorf("error forking repo: %w", err)
 		}
 		didForkRepo = true
+	}
+
+	headBranchLabel := headBranch
+	if !ghrepo.IsSame(baseRepo, headRepo) {
+		headBranchLabel = fmt.Sprintf("%s:%s", headRepo.RepoOwner(), headBranch)
+	}
+
+	headRemote, err := repoContext.RemoteForRepo(headRepo)
+	// There are two cases when an existing remote for the head repo will be
+	// missing:
+	// 1. the head repo was just created by auto-forking;
+	// 2. an existing fork was discovered by quering the API.
+	//
+	// In either case, we want to add the head repo as a new git remote so we
+	// can push to it.
+	if err != nil {
 		// TODO: support non-HTTPS git remote URLs
-		baseRepoURL := fmt.Sprintf("https://github.com/%s.git", ghrepo.FullName(baseRepo))
 		headRepoURL := fmt.Sprintf("https://github.com/%s.git", ghrepo.FullName(headRepo))
-		// TODO: figure out what to name the new git remote
-		gitRemote, err := git.AddRemote("fork", baseRepoURL, headRepoURL)
+		// TODO: prevent clashes with another remote of a same name
+		gitRemote, err := git.AddRemote("fork", headRepoURL, "")
```

Comment:
as of this pull request, the third argument of `AddRemote` is unused anywhere. Do we still need to support it?

Reply:
Good call! I've simplified the AddRemote implementation